database/src/bin/import-sqlite.rs

```diff
@@ -41,7 +41,7 @@ async fn main() {
         for &(benchmark, profile, scenario, metric) in sqlite_idx.all_statistic_descriptions() {
             if benchmarks.insert(benchmark) {
                 postgres_conn
-                    .record_benchmark(benchmark.as_str(), None)
+                    .record_benchmark(benchmark.as_str(), None, "") // TODO
```

Comment:
I think this probably wants to be a meaningful string -- secondary.

Reply:
Well mainly this should be copied from SQLite to Postgre, right? Is this import being used anywhere, is it important?